UIKitTask/Sources/Scenes/PostsScene/PostsViewController.swift

```diff
@@ -0,0 +1,184 @@
+// PostsViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Стартовый экран, список постов , сторис, рекомендаций
+final class PostsViewController: UIViewController {
+    // MARK: Types
+
+    private enum TableSections {
+        case firstPost
```

Comment:
```suggestion
        case firstPost
```
доку на экнам и все кейсы